src/modules/communication/SerialConsole.cpp

```diff
@@ -21,7 +21,7 @@ using std::string;
 // Treats every received line as a command and passes it ( via event call ) to the command dispatcher.
 // The command dispatcher will then ask other modules if they can do something with it
 SerialConsole::SerialConsole( PinName rx_pin, PinName tx_pin, int baud_rate ){
-    this->serial = new mbed::Serial( rx_pin, tx_pin );
+    this->serial = new mbed::Serial( tx_pin, rx_pin );
```

Comment:
The correct fix here is to flip the rx_pin/tx_pin parameters in the SerialConsole() constructor since that is the order the existing code calling the constructor are actually passing in.